bower.json

```diff
@@ -2,19 +2,20 @@
   "name": "styleguide",
   "version": "0.1.0",
   "dependencies": {
+    "highlightjs": "^9.2.0",
     "angular": "1.5.5",
     "angular-animate": "1.5.5",
-    "angular-highlightjs": "0.3.0",
-    "angular-mocks": "1.5.5",
-    "angular-bootstrap-colorpicker": "3.0.8",
-    "angular-local-storage": "0.1.0",
-    "ui-router": "0.2.13",
-    "oclazyload": "0.3.8",
-    "ngprogress": "~1.0.7",
-    "angular-debounce": "~1.0.0",
-    "angular-scroll": "0.7.0",
-    "lodash": "~3.9.3",
-    "custom-event": "~0.1.0"
+    "angular-mocks": "^1.5.5",
```

Comment:
I noticed that you have fixed your versions to specific version numbers. However, I've used the caret notation to fix packages to the 1.x versions. I could easily pull those back out if that's your preference.